lib/matplotlib/collections.py

```diff
@@ -1262,62 +1262,60 @@ def __init__(self,
+        linelength : scalar, optional. Default is 1.
+            The total height of the marker (i.e. the marker stretches from
+            ``lineoffset + linelength/2`` to ``lineoffset - linelength/2``).
 
-        *linewidth*
-            a single numerical value
+        linewidth : scalar or None, optional. Default is None.
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *color*
-            must be a sequence of RGBA tuples (e.g., arbitrary color
-            strings, etc, not allowed).
+        color : color, sequence of colors or None, optional. Default is None.
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *linestyle* [ 'solid' | 'dashed' | 'dashdot' | 'dotted' ]
+        linestyle : str or tuple, optional. Default is 'solid'.
+            If it is a string, it has to be among [ 'solid' | 'dashed' |
+            'dashdot' | 'dotted' | '-' | '--' | '-.' | ':' ]. If it is a dash
+            tuple, it has to be of the form::
 
-        *antialiased*
-            1 or 2
+                (offset, onoffseq),
 
-        If *linewidth*, *color*, or *antialiased* is None, they
-        default to their rcParams setting, in sequence form.
+            where *onoffseq* is an even length tuple of on and off ink
+            in points.
 
-        *norm*
-            None (optional for :class:`matplotlib.cm.ScalarMappable`)
-        *cmap*
-            None (optional for :class:`matplotlib.cm.ScalarMappable`)
+        antialiased : [None (default) | 1 | 2], optional.
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *pickradius* is the tolerance for mouse clicks picking a line.
-        The default is 5 pt.
+        Additional kwargs of :class:`~matplotlib.collections.LineCollection`
+        can also be used:
 
-        The use of :class:`~matplotlib.cm.ScalarMappable` is optional.
-        If the :class:`~matplotlib.cm.ScalarMappable` array
-        :attr:`~matplotlib.cm.ScalarMappable._A` is not None (i.e., a call to
-        :meth:`~matplotlib.cm.ScalarMappable.set_array` has been made), at
-        draw time a call to scalar mappable will be made to set the colors.
+        %(LineCollection)s
 
-        **Example:**
+        Example
+        -------
 
         .. plot:: mpl_examples/pylab_examples/eventcollection_demo.py
         """
 
         segment = (lineoffset + linelength / 2.,
                    lineoffset - linelength / 2.)
-        if len(positions) == 0:
+        if positions is None or len(positions) == 0:
             segments = []
         elif hasattr(positions, 'ndim') and positions.ndim > 1:
-            raise ValueError('if positions is an ndarry it cannot have '
-                             'dimensionality great than 1 ')
+            raise ValueError('if positions is an ndarray it cannot have '
+                             'dimensionality strictly greater than 1 ')
```

Comment:
https://en.wikipedia.org/wiki/Inequality_(mathematics)
https://en.wikipedia.org/wiki/Sign_(mathematics)#non-negative_and_non-positive
"Because zero is neither positive nor negative (in most countries)" I guess we just like being different in France...